persistence.lua

```diff
@@ -21,7 +21,11 @@ function travelnet.restore_data()
 	end
 
 	local data = file:read("*all")
-	travelnet.targets = minetest.deserialize(data)
+	if data:sub(1, 1) == "[" then
```

Comment:
It looks like `travelnet.targets` isn't a list so the JSON data will start wtih `{` and not `[`.

```suggestion
	if data:sub(1, 1) == "{" then
```